bindings/java/iota-wallet-java/src/test/java/org/iota/SimpleTest.java

```diff
@@ -0,0 +1,60 @@
+package org.iota;
+
+import org.iota.types.AccountAddress;
+import org.iota.types.AccountHandle;
+import org.iota.types.account_methods.SetAlias;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.ids.account.AccountAlias;
+import org.iota.types.ids.account.AccountIndex;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SimpleTest extends ApiTest {
+
+    @Test
+    public void testCreateAccount() throws WalletException {
+        System.out.println(wallet.createAccount("Hans"));
+    }
+
+    @Test
+    public void testGetAccountByAlias() throws WalletException {
+        AccountHandle a = wallet.createAccount("Hans");
+        AccountHandle b = wallet.getAccount(new AccountAlias("Hans"));
```

Comment:
Can the tests create an account with the same alias? If it's the same account manager, it shouldn't work

Reply:
Added

Reply:
No I meant, do the tests use the same account manager? Like are the tests then actually working and doesn't one test create it in the same db as the next one would do?

Reply:
After each test, all accounts are deleted. So the tests start up with a fresh state